build/build-cdt-lib.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const fs = require('fs');
+
+const outDir = `${__dirname}/../lighthouse-core/lib/cdt/generated`;
+const files = {
+  'node_modules/chrome-devtools-frontend/front_end/sdk/SourceMap.js': 'SourceMap.js',
+};
+
+// eslint-disable-next-line no-console
+console.log('making modifications ...');
+
+for (const [input, output] of Object.entries(files)) {
```

Comment:
previously i had a couple files here, but I got it down to just the one. Left it around in case another file must be added - if that doesn't come up in code review soon then I'll remove it

Reply:
![image](https://user-images.githubusercontent.com/4071474/71026170-7f028200-20bd-11ea-9511-64c505e6d4b9.png)
not often. most changes are from recent frontend refactors.